src/components/KonnectorInstall.jsx

```diff
@@ -9,9 +9,13 @@ import LegacyKonnectorInstall from 'components/LegacyKonnectorInstall'
 import { getKonnector } from 'ducks/konnectors'
 import styles from 'styles/konnectorInstall'
 
-export class KonnectorInstall extends PureComponent {
+export class KonnectorInstall extends Component {
```

Comment:
KonnectorInstall can still be a `PureComponent`, I see no reason to change it actually. Do you see any reason ?

Reply:
https://github.com/cozy/cozy-home/pull/1256#discussion_r294691453